blueflood-elasticsearch/src/main/resources/events_mapping.json

```diff
@@ -0,0 +1,27 @@
+{
+  "graphite_event": {
+    "_routing": {
+      "required": true
+    },
+    "properties": {
+      "tenantId": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "what": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "data": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "tags": {
+        "type": "string"
```

Comment:
@chinmay-gupte not analyzed means that it wont be tokenized and will be compared as is. It's still searchable though. I think that's fine. What do you think?